pkg/cli/cli.go

```diff
@@ -15,7 +15,6 @@ import (
 	"time"
 
 	"github.com/blang/semver"
-	"github.com/exercism/cli/debug"
```

Comment:
I like that we don't have the debug package here anymore. That feels cleaner.

Reply:
Yeah me too. It had one odd sprinkle in the previous cli pkg. I like having the option to create a custom API client with all the bells and whistles that we may want.